src/main/java/net/minecraftforge/fml/client/GuiModList.java

```diff
@@ -67,11 +67,21 @@
 import org.lwjgl.opengl.GL11;
 
 /**
- * @author cpw
+ * This class represents the GUI screen for the mod list.
+ * It extends the GuiScreen class provided by Minecraft Forge.
```

Comment:
same strange

Reply:
> What do you mean?

You do not need to tell modder what it entends.

Reply:
Fixed